docs/dptp-triage-sop/logs/warnings.txt

```diff
@@ -0,0 +1,17 @@
+ fields @message
+| sort @timestamp desc 
+| filter(level = "warning")
+| filter(not(
+# exclusions we expect to keep forever
+	component="entrypoint" or component="sidecar" or component="clonerefs" or component="initupload" # these are logs from jobs, which we don't care about
+	or `kubernetes.pod_labels.prow.k8s.io/type`="presubmit" # these are logs from jobs, which we don't care about
+	or msg="Throttled clientside for more than 15 minutes" # this is just token contention
```

Comment:
I'm not convinced this is an exclusion: throttling for 15+ m can cause some serious trouble and sometimes means we should give more tokens to something.

Reply:
I thought we throttled because we had no more tokens to give?

Reply:
Nope, the client-side throttling when enabled gives a hourly budget to the client of how many tokens it can burn. If you routinely burn more for a long period of time, then in some cases (needs-rebase) you may want to increase the budget -or at least consider it. 15m is a noticeable delay and probably means you are burning much more than your settings are.

Reply:
What about we card a jira issue to think about the client-side throttling (hiding it sometimes? Allow plugins to provide timeouts for their requests?) and then changing the query?